src/components/NestedField.js

```diff
@@ -59,25 +62,41 @@ class NestedField extends React.Component {
       getFullField: field => [fullField, field]
     }
 
+    const proxySubField = method => (subField, ...args) =>
+      method([fullField, subField].filter(Boolean), ...args)
```

Comment:
Can you explain this? we never needed subField before? where does that come from. I thought that was the job of overidding the getFullField on line 62 above?

Reply:
The reason this is necessary is because the api for a NestedField should resemble that of a root form, not a field. Since we don't have any areas where we actually use the form api from a nested field, we missed that. This makes interacting with the nested field api actually useful.
```javascript
<NestedField field='profile' render={(api => (
  <button onClick={() => api.setValue('name', 'Tanner')} />
)} />
```
or
```javascript
<NestedField field='users' defaultValue={[]} render={(api => (
  <button onClick={() => api.addValue(undefined, {...} )}>Add User</button>
)} />
```

Reply:
Ahh ok cool